rubocop.yml

```diff
@@ -165,12 +165,11 @@ Naming/FileName:
   Regex:
   IgnoreExecutableScripts: true
 
-Layout/FirstArgumentIndentation:
-  EnforcedStyle: consistent
+Layout/ArgumentAlignment:
+  EnforcedStyle: with_fixed_indentation
   SupportedStyles:
-  - consistent
-  - special_for_inner_method_call
-  - special_for_inner_method_call_in_parentheses
+    - with_first_argument
+    - with_fixed_indentation
   IndentationWidth:
```

Comment:
Maybe we can remove this

```suggestion
```

Reply:
Quite a few rules have it empty, but [looks like empty (nil) is the same as not specifying](https://github.com/rubocop-hq/rubocop/blob/48c1e7a9e3b34ae145f4c20785a1bd04c7ec91d6/lib/rubocop/cop/mixin/alignment.rb#L14-L17). 👍